experiment/build/test_builder.py

```diff
@@ -24,7 +24,7 @@
 
 SRC_ROOT = os.path.join(os.path.dirname(__file__), os.pardir, os.pardir)
 
-FUZZER_BLACKLIST = {'coverage'}
+FUZZER_BLACKLIST = {'coverage', 'coverage_source_based'}
```

Comment:
Consider using `COVERAGE_TOOLS` or `FUZZER_SKIP_LIST`

Reply:
Done it. Also updated its name in other files.